### ISSUE TITLE
ACCESSIBILITY TWEAKS/BUGS

### ISSUE THREAD

**Issue by melxincognito** [melxincognito/kp_jewelry#15]
Using Screen Reader:

**MAIN LANDING PAGE**
- When first landing on the main landing page it immediately announces 'skip to main content' twice which seems buggy. Fix to announce the page name as well. 

- It goes through the items but if someones blind it's going to be confusing because for the category buttons it announces what they are ex 'link, necklaces' but I feel like it might not be obvious it's a section where you're able to see items by category so fix this. 

**ITEMS CATALOG**
- I dont think it's obvious what the buttons are for ex if this is a filter button and how you have to go through all the filter options to get to the main content section. Maybe they just want to get to the items or pick one filter that's really inconvenient. 

**INDIVIDUAL ITEM PAGE**
- When going onto an item it doesn't announce anything on the page so it seems pretty useless, fix it so it lets them know the description and everything. 

**SIGN IN PAGE**
- Also doesn't really announce what the page is for just the field items and that it's KP Jewelers. Email voiceover is a little confusing, check the wording. Password field is fine. The create an account because they don't have one option button is also confusing it just says 'Create one' on voiceover so if someone's blind they're going to think what is this for it's not obvious. 

**CREATE ACCOUNT PAGE**
- Again doesn't clearly state what page this is for. First email field voiceover text is also a little confusing but the other fields seem to be ok. The button to sign in because you already have an account doesn't seem obvious as to what it's for either fix that.

**Comment by melxincognito** [comment]
[Semantics Cheat Sheet](https://webaim.org/resources/htmlcheatsheet/HTML%20Semantics%20and%20Accessibility%20Cheat%20Sheet.pdf)

[React Accessibility and Semantics](https://reacthandbook.dev/semantics)

[Accessibility Cheat Sheet](https://learntheweb.courses/topics/accessibility-cheat-sheet/)
